packages/pci-sdk-iframe/src/pages/app/themes/dark.ts

```diff
@@ -86,4 +86,20 @@ export default {
 		boxSizing: 'border-box',
 		margin: '0',
 	},
+	// Notification
+	notification: {
+		fontFamily: 'sans-serif',
+		padding: '2vw 4vw',
+		lineHeight: '24px',
+		position: 'absolute',
+		left: '50%',
+		transform: 'translateX(-50%)',
+		backgroundColor: '#263238',
+		color: '#CFD8DC',
+		borderRadius: '6px',
+		zIndex: 3,
+		width: '70%',
+		textAlign: 'center',
+		top: '15px',
```

Comment:
I would change the CSS a bit if you don't mind, so we keep using relative units in those rules that need it and better follow the styleguide guidelines for branding consistency.

```suggestion
	position: 'absolute',
	zIndex: 3,
	top: 4.706vw;
	left: '50%',
	width: '70%',
	padding: '2vw 4vw',
	transform: 'translateX(-50%)',	
	borderRadius: '8px',
	backgroundColor: '#333333',
	color: '#CFD8DC',
	fontFamily: 'sans-serif',
	font-size: 4.118vw;
  line-height: 1.5;
  textAlign: 'center',
```